dts/arm/ti/am64x_m4.dtsi

```diff
@@ -29,88 +30,52 @@
 		reg = <0x0 DT_SIZE_K(192)>;		/* 192 KB of SRAM (I-Code) */
 	};
 
-	sram1: memory1@40000 {
+	sram1: memory1@30000 {
```

Comment:
This looks correct, but it is a fix unrelated to this refactor. Should this go in its own commit?